src/pal/src/exception/seh-unwind.cpp

```diff
@@ -223,6 +223,11 @@ RaiseException(IN DWORD dwExceptionCode,
         dwExceptionCode ^= RESERVED_SEH_BIT;
     }
 
+    if (nNumberOfArguments > EXCEPTION_MAXIMUM_PARAMETERS) 
```

Comment:
There is the same check a few lines below which won't be needed with this change. Could you please remove it?